cinema/urls.py

```diff
@@ -0,0 +1,10 @@
+from django.urls import path
+
+from cinema.views import get_or_create_movie, get_update_or_delete_movie
+
+app_name = "cinema"
+
+urlpatterns = [
+    path("movies/", get_or_create_movie, name="movies"),
```

Comment:
movies / movie are very general names. Can you be more specific there?

Reply:
name="movies"
name="movie"

When you use these kind of names in redirects or other references it will be very easy to get confused. So I meant to change them to something meaningful - like get_create_movie. This way if you see it in code somewhere you will know exactly what it does